app/controllers/events_controller.rb

```diff
@@ -19,8 +19,7 @@ def archives
 
   def show
     @timecards = Timecard.active.overlapping_time(@event.start_time..@event.end_time)
-    @responses_and_assignments = ( @event.responses +
-       Assignment.active.for_time_span(@event.start_time..@event.end_time)).to_a
+    @responses_and_assignments = @event.responses + @event.assignments
```

Comment:
@kgf Ok, so you would like it to show all assignments within that events time frame?

Reply:
Yes, but that can be a different PR.
For those playing along at home, there is a use case for this. If you have someone showing as available for a time period, that will show on all events in that period. But you may have given them an assignment in Event B. If I'm doing assignments in Event A, I would think I could give them an assignment; they're available, but they're really not. I do not want to remove their availability from the list because I might think the assignment in Event A is more important, and pull them from Event B.